kotlin/internal/jvm/impl.bzl

```diff
@@ -243,11 +243,13 @@ def kt_jvm_library_impl(ctx):
 
 def kt_jvm_binary_impl(ctx):
     providers = _kt_jvm_produce_jar_actions(ctx, "kt_jvm_binary")
+    jvm_flags = ctx.attr.jvm_flags[:]
+    jvm_flags.extend(ctx.fragments.java.default_jvm_opts)
```

Comment:
Will ordering be an issue here, meaning jvm_flags that are specific to the `kt_jvm_binary` target can be overridden by the default jvm opts provided by the java fragment?